tests/_support/Page/Widget/Grid.php

```diff
@@ -140,6 +140,20 @@ public function getColumnNumber(string $columnName): int
         throw new \Exception("failed detect column with name $columnName");
     }
 
+    public function getRowNumberInColumnByValue(int $column, string $rowValue): int
```

Comment:
```suggestion
    public function getRowNumberInColumnByValue(string $columnName, string $rowValue): int
```
ти коли використовуєш це писав `$col = $index->gridView->getColumnNumber()`
пропоную пошук айді колонки засунуть сюда

Reply:
колонка не имеет айди жеж? или ты имеешь ввиду название колонки?